andela/ah-centauri-frontend
# Chore(display toast messages): Users can see toast messages

#### What does this PR do?
Allows the toast messages to be displayed and give users feedback on their interactions

#### Description of Tasks to be completed?
Users should be able to get feedback based on interactions with the application and see errors when there are issues carrying out their request.

#### How should this be manually tested?
- Clone the project: git clone https://github.com/andela/ah-centauri-frontend
- cd into the project folder
- checkout to branch `chore/1661157320-display-toast-messages `
- Install dependencies: npm install
- Launch development server: npm dev
- navigate to the homepage page
- Sign out and attempt to access the profile page at `/profile` endpoint, a toast message should display with details on why the request could not be completed.

#### Any background context you want to provide?
Toast messages have only been added to the profile page, the component is loaded inside the HeaderLayout file. Should a page want to display the toast message based on server feedback the page must have the HeaderLayout or import the SemanticToastContainer component into their render method before using the `setToastMessage` function found in the errorMessages utilities.

#### What are the relevant Pivotal Tracker Stories?
[#1661157320](https://www.pivotaltracker.com/story/show/1661157320)

#### Screenshots(if appropriate)
![image](https://user-images.githubusercontent.com/31648893/58082551-e5730080-7bbf-11e9-9920-51c8439f84ed.png)

![image](https://user-images.githubusercontent.com/31648893/58082492-caa08c00-7bbf-11e9-88f5-8c3cbc254b3f.png)
